ui/app/adapters/config-ui/message.js

```diff
@@ -11,8 +11,11 @@ export default class MessageAdapter extends ApplicationAdapter {
   }
 
   query(store, type, query) {
-    const { authenticated } = query;
-    return super.query(store, type, { authenticated, list: true });
+    const { authenticated, type: messageType, active } = query;
+    const params = { authenticated, list: true };
+    if (messageType) params.type = messageType;
+    if (typeof active === 'boolean' && typeof active !== 'undefined') params.active = active;
```

Comment:
Doing these checks since active can be false or undefined which are falsy values. We only want to see active to true or false.

Reply:
We only need one of these checks -- if active's type is `boolean` it will by definition not be type `undefined`
```suggestion
    if (typeof active === 'boolean') params.active = active;
```